site/content/docs/5.2/components/modal.md

```diff
@@ -699,10 +713,18 @@ Another override is the option to pop up a modal that covers the user viewport,
 | `.modal-fullscreen-sm-down` | `480px` |
 | `.modal-fullscreen-md-down` | `768px` |
 | `.modal-fullscreen-lg-down` | `1024px` |
-| `.modal-fullscreen-xl-down` | `1280px` |
+| `.modal-fullscreen-xl-down` | `1280px` |s
 | `.modal-fullscreen-xxl-down` | `1440px` |
 {{< /bs-table >}}
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+These variants should not be used because they do not respect the Orange Design System specifications.
```

Comment:
Maybe that's because I've read dozens of callouts here but IMO having the reason mentioned could help. It's not always that obvious.

Reply:
OK. What about:
> These **full screen** variants should not be used because they do not respect the Orange Design System specifications. Indeed, modals should always be placed in the center of a page and not be full screen.
> Please refer to the [Modals](https://system.design.orange.com/0c1af118d/p/16d9f3-modals/b/774d3d) guidelines on the Orange Design System website.

Reply:
Resolved ?